src/knowledge_sources/StreamSegregationKS.m

```diff
@@ -109,8 +109,10 @@ function execute(obj)
                 end
             else
                 locHypos = obj.blackboard.getLastData( 'locationHypothesis' );
+                isLocHyp = true;
```

Comment:
Really knitpicking, but the following lines can be cleaner like

isLocHyp = ~isempty( locHypos );
if ~isLocHyp
    locHypos = getLast sourcesAzmDist Data
end